src/html/html-element-reference-scanner.ts

```diff
@@ -58,7 +58,18 @@ export class HtmlElementReferenceScanner implements HtmlScanner {
 
         elements.push(element);
       }
-    });
+
+      // Descend into templates.
+      if (node.tagName === 'template' && node['content']) {
```

Comment:
Yeah, I mean, looking at the other HTML scanners, I think this is the only one that actually does want to descend into templates. Most of the others wouldn't really ever see any use inside of templates so it doesn't matter much, but it does hint at which way the default should point.